src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Players/PlayerViewModel.cs

```diff
@@ -115,16 +117,40 @@ public static PlayerViewModel Map(Player player)
         /// <returns> Domain object </returns>
         public Player ToDomain()
         {
-            return new Player {
-                Id = Id,
-                FirstName = FirstName,
-                LastName = LastName,
-                BirthYear = BirthYear,
-                Height = Height,
-                Weight = Weight,
-                TeamId = TeamId
+            return new Player(Id, FirstName, LastName, BirthYear, Height, Weight);
+        }
+
+        /// <summary>
+        /// Maps presentation entity to CreatePlayerDto
+        /// </summary>
+        /// <returns> CreatePlayerDto object </returns>
+        public CreatePlayerDto ToCreatePlayerDto()
+        {
+            return new CreatePlayerDto {
+                FirstName = this.FirstName,
+                LastName = this.LastName,
+                BirthYear = this.BirthYear,
+                Height = this.Height,
+                Weight = this.Weight
             };
         }
+
         #endregion
-    }
+
+        public FreePlayerDto ToFreePlayerDto()
+        {
+            if (this.TeamId == null)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the '?:' operator here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3240)